modules/core/src/smithy4s/StaticService.scala

```diff
@@ -0,0 +1,37 @@
+package smithy4s
+
+import smithy4s.kinds.FunctorK5
+import smithy4s.kinds.PolyFunction5
+
+trait StaticService[P[_[_, _, _, _, _]]] extends FunctorK5[P] {
+  type Alg[_[_, _, _, _, _]]
+  val service: Service[Alg]
+  def endpoints: P[service.Endpoint]
+
+  def toPolyFunction[P2[_, _, _, _, _]](
+      algebra: P[P2]
+  ): PolyFunction5[service.Endpoint, P2]
+}
+
+object StaticService {
+
+  type Aux[Alg[_[_, _, _, _, _]], P[_[_, _, _, _, _]]] =
+    StaticService[Alg] {
+      type Alg[Op[_, _, _, _, _]] = P[Op]
+    }
```

Comment:
This needs to align with the rest : 

```suggestion
  type Aux[Prod[_[_, _, _, _, _]], Algebra[_[_, _, _, _, _]]] =
    ServiceProduct[Prod] {
      type Alg[F[_, _, _, _, _]] = Algebra[F]
    }
```

Reply:
think I messed this , fixed at [a2e82b7](https://github.com/disneystreaming/smithy4s/pull/1011/commits/a2e82b706cd31ecbd3ba9304fab6ad2dadb79f46)